memcached.c

```diff
@@ -5552,6 +5622,41 @@ static void process_misbehave_command(conn *c) {
         out_string(c, "OK");
     }
 }
+
+static void process_debugtime_command(conn *c, token_t *tokens, const size_t ntokens) {
+    struct timeval tv;
+
+    if (strcmp(tokens[1].value, "p") == 0) {
+        gettimeofday(&tv, NULL);
+        pthread_mutex_lock(&debugtime_lock);
+        if (!is_paused) {
+            paused_time = (rel_time_t)tv.tv_sec;
```

Comment:
tracking time spent paused is clever but I think unnecessary. or at least, I can't think of a use case for it and it adds a bit to this code here.